unit/test_tags.py

```diff
@@ -45,6 +45,7 @@ def test_without_tags(self):
             'Set remote_user for delegated tasks',
             'Validate config',
             'Set instance facts',
+            'Calculate machine hosts list',
```

Comment:
Get physical machines list?
Get one instance from each physical machine to install package?

Reply:
Get one instance from each physical machine